keyboards/jasonhazel/bad_wings/keymaps/default/config.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include "config_common.h"
+
+#define TAPPING_TERM 200
+#define PERMISSIVE_HOLD
+#define IGNORE_MOD_TAP_INTERRUPT
+#define TAPPING_FORCE_HOLD
```

Comment:
`TAPPING_FORCE_HOLD` feature is now replaced by `QUICK_TAP_TERM`. See:
https://docs.qmk.fm/#/ChangeLog/20230226?id=quick-tap-term